exercises/Scientific-2-R.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: exercise
+title: Scientific 2
+language: R
+---
+
+This is a follow up to
+[Scientific 1]({{ site.baseurl }}/exercises/Scientific-1-R).
+
+Looking at the average mass of extinct and extant species overall is useful, but
+there are lots of different processes that could cause size-biased extinctions
+so it's not as informative as we might like.  However, if we see the exact same
+pattern on each of the different continents that might really tell us
+something. Repeat the analysis in
+[Scientific 1]({{ site.baseurl }}/exercises/Scientific-1-R), but this time compare the
+mean masses within each of the different continents. Export your results to a
+csv file where the first entry on each line is the continent, the second entry
+is the average mass of the extant species on that continent, the third entry is
+the average mass of the extinct species on that continent, and the forth entry
+is the difference between the average extant and average extinct masses. Call
+the file `continent_mass_differences.csv`. If you notice anything
+strange think about what's going on and present the final data in the way that
+makes the most sense to you.
+
+Because you’re working with real data, some things might look strange that are
+not results but barriers to analysis. These inconsistencies in the data will 
+require some cleaning. 
+
+The unknown value used in the dataset is `-999`. R assumes your unknown value is 
+`NA`, but `“NA”` in the data is the code for North America. 
```

Comment:
> you want me to remove the back quotes everywhere

No, not the back quotes. You currently have `“NA”`, but the double quotes in here are not regular double quotes, and they will error if you enter them into R:

```
> “NA”
Error: unexpected input in "�"
```

They need to be replaced with regular double quotes: `"NA"`